File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.py
class Solution:
    def minDeletionSize(self, strs):
        remove = 0
        for i in range(len(strs[0])):
            for j in range(len(strs)-1):
                if(strs[j][i] > strs[j+1][i]):
                    remove += 1
                    break
        return remove 
        

File: 0944-delete-columns-to-make-sorted/0944-delete-columns-to-make-sorted.cpp
class Solution {
public:
    int minDeletionSize(vector<string>& strs) {
    int remove = 0;
    for(int i=0 ; i<strs[0].size();++i)
        for(int j=0;j<strs.size()-1;++j)
            if(strs[j][i] > strs[j+1][i]){
                ++remove;
                break;
            }
    return remove;
    }
};